vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SpaceManagerTest.java

```diff
@@ -116,23 +125,32 @@ public void test_storeSpace_success() throws FileStorageException {
         Mockito.verify(spaceDisplayRepo).save((SpaceDisplay)spaceDisplay);
     }
 
-
-
-    @Test(expected = SpaceDoesNotExistException.class)
-    public void test_deleteSpaceById_whenIdIsNull() throws SpaceDoesNotExistException {
-        Mockito.doThrow(IllegalArgumentException.class)
-        .when(spaceRepo).deleteById(null);
+    @Test
+    public void test_deleteSpaceById_whenIdIsNull() throws SpaceDoesNotExistException{
+        Mockito.when(managerToTest.getSpace(spaceId)).thenReturn(null).thenReturn(null);
+        Assert.assertEquals(null, managerToTest.getSpace(null));
         managerToTest.deleteSpaceById(null);
+        Assert.assertEquals(null, managerToTest.getSpace(null));
     }
 
-    @Test(expected = SpaceDoesNotExistException.class)
-    public void test_deleteSpaceById_forNonExistentId() throws SpaceDoesNotExistException {
-        String imgId = "imgId";
-        Mockito.doThrow(EmptyResultDataAccessException.class)
-        .when(spaceRepo).deleteById(imgId);
-        managerToTest.deleteSpaceById(imgId);
+    @Test
+    public void test_deleteSpaceById_forNonExistentId() {  
+        Mockito.when(managerToTest.getSpace(spaceId)).thenReturn(null).thenReturn(null);
+        Assert.assertEquals(null, managerToTest.getSpace(spaceId));
+        managerToTest.deleteSpaceById(spaceId);
+        Assert.assertEquals(null, managerToTest.getSpace(spaceId));
```

Comment:
this test doesn't make sense. What's the expected outcome of calling `deleteSpaceById` with a non-existing id? You should only test that expectation.